source/features/avoid-accidental-submissions.tsx

```diff
@@ -1,40 +1,61 @@
+import React from 'dom-chef';
 import select from 'select-dom';
 import delegate from 'delegate-it';
 import * as pageDetect from 'github-url-detection';
 
+import {isMac} from '../github-helpers';
+
```

Comment:
Extra Space

Reply:
Ok, I swapped its place with the import after it, and removed the extra line.